Description: Upstream changes introduced in version 0.4.3-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 pykolab (0.4.3-1) unstable; urgency=low
 .
   * Initial version
 .
 The person named in the Author field signed this changelog entry.
Author: Jeroen van Meeuwen (Kolab Systems) <vanmeeuwen@dbs02.kolabsys.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ pykolab-0.4.3/config.log
@@ -0,0 +1,526 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by pykolab configure 0.4.3, which was
+generated by GNU Autoconf 2.68.  Invocation command line was
+
+  $ ./configure --build=x86_64-linux-gnu --prefix=/usr --includedir=${prefix}/include --mandir=${prefix}/share/man --infodir=${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --libexecdir=${prefix}/lib/pykolab --disable-maintainer-mode --disable-dependency-tracking
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = dbs02
+uname -m = x86_64
+uname -r = 2.6.32-5-amd64
+uname -s = Linux
+uname -v = #1 SMP Wed Jan 12 03:40:32 UTC 2011
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/sbin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /bin
+PATH: /usr/bin/X11
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2236: checking for a BSD-compatible install
+configure:2304: result: /usr/bin/install -c
+configure:2315: checking whether build environment is sane
+configure:2365: result: yes
+configure:2506: checking for a thread-safe mkdir -p
+configure:2545: result: /bin/mkdir -p
+configure:2558: checking for gawk
+configure:2574: found /usr/bin/gawk
+configure:2585: result: gawk
+configure:2596: checking whether make sets $(MAKE)
+configure:2618: result: yes
+configure:2699: checking whether to enable maintainer-specific portions of Makefiles
+configure:2708: result: no
+configure:2733: checking for python
+configure:2751: found /usr/bin/python
+configure:2763: result: /usr/bin/python
+configure:2784: checking for python version
+configure:2791: result: 2.6
+configure:2803: checking for python platform
+configure:2810: result: linux2
+configure:2817: checking for python script directory
+configure:2845: result: ${prefix}/lib/python2.6/dist-packages
+configure:2854: checking for python extension module directory
+configure:2882: result: ${exec_prefix}/lib/python2.6/dist-packages
+configure:2910: checking for style of include used by make
+configure:2938: result: GNU
+configure:3009: checking for gcc
+configure:3025: found /usr/bin/gcc
+configure:3036: result: gcc
+configure:3265: checking for C compiler version
+configure:3274: gcc --version >&5
+gcc-4.4.real (Debian 4.4.5-8) 4.4.5
+Copyright (C) 2010 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:3285: $? = 0
+configure:3274: gcc -v >&5
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.5-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.4 --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.5 (Debian 4.4.5-8) 
+configure:3285: $? = 0
+configure:3274: gcc -V >&5
+gcc-4.4.real: '-V' option must have argument
+configure:3285: $? = 1
+configure:3274: gcc -qversion >&5
+gcc-4.4.real: unrecognized option '-qversion'
+gcc-4.4.real: no input files
+configure:3285: $? = 1
+configure:3305: checking whether the C compiler works
+configure:3327: gcc -g -O2   conftest.c  >&5
+configure:3331: $? = 0
+configure:3379: result: yes
+configure:3382: checking for C compiler default output file name
+configure:3384: result: a.out
+configure:3390: checking for suffix of executables
+configure:3397: gcc -o conftest -g -O2   conftest.c  >&5
+configure:3401: $? = 0
+configure:3423: result: 
+configure:3445: checking whether we are cross compiling
+configure:3453: gcc -o conftest -g -O2   conftest.c  >&5
+configure:3457: $? = 0
+configure:3464: ./conftest
+configure:3468: $? = 0
+configure:3483: result: no
+configure:3488: checking for suffix of object files
+configure:3510: gcc -c -g -O2  conftest.c >&5
+configure:3514: $? = 0
+configure:3535: result: o
+configure:3539: checking whether we are using the GNU C compiler
+configure:3558: gcc -c -g -O2  conftest.c >&5
+configure:3558: $? = 0
+configure:3567: result: yes
+configure:3576: checking whether gcc accepts -g
+configure:3596: gcc -c -g  conftest.c >&5
+configure:3596: $? = 0
+configure:3637: result: yes
+configure:3654: checking for gcc option to accept ISO C89
+configure:3718: gcc  -c -g -O2  conftest.c >&5
+configure:3718: $? = 0
+configure:3731: result: none needed
+configure:3753: checking dependency style of gcc
+configure:3864: result: none
+configure:3885: checking how to run the C preprocessor
+configure:3916: gcc -E  conftest.c
+configure:3916: $? = 0
+configure:3930: gcc -E  conftest.c
+conftest.c:11:28: error: ac_nonexistent.h: No such file or directory
+configure:3930: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "pykolab"
+| #define PACKAGE_TARNAME "pykolab"
+| #define PACKAGE_VERSION "0.4.3"
+| #define PACKAGE_STRING "pykolab 0.4.3"
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "pykolab"
+| #define VERSION "0.4.3"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:3955: result: gcc -E
+configure:3975: gcc -E  conftest.c
+configure:3975: $? = 0
+configure:3989: gcc -E  conftest.c
+conftest.c:11:28: error: ac_nonexistent.h: No such file or directory
+configure:3989: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME "pykolab"
+| #define PACKAGE_TARNAME "pykolab"
+| #define PACKAGE_VERSION "0.4.3"
+| #define PACKAGE_STRING "pykolab 0.4.3"
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define PACKAGE "pykolab"
+| #define VERSION "0.4.3"
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:4018: checking for grep that handles long lines and -e
+configure:4076: result: /bin/grep
+configure:4081: checking for egrep
+configure:4143: result: /bin/grep -E
+configure:4148: checking for ANSI C header files
+configure:4168: gcc -c -g -O2  conftest.c >&5
+configure:4168: $? = 0
+configure:4241: gcc -o conftest -g -O2   conftest.c  >&5
+configure:4241: $? = 0
+configure:4241: ./conftest
+configure:4241: $? = 0
+configure:4252: result: yes
+configure:4265: checking for sys/types.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for sys/stat.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for stdlib.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for string.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for memory.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for strings.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for inttypes.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for stdint.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4265: checking for unistd.h
+configure:4265: gcc -c -g -O2  conftest.c >&5
+configure:4265: $? = 0
+configure:4265: result: yes
+configure:4280: checking locale.h usability
+configure:4280: gcc -c -g -O2  conftest.c >&5
+configure:4280: $? = 0
+configure:4280: result: yes
+configure:4280: checking locale.h presence
+configure:4280: gcc -E  conftest.c
+configure:4280: $? = 0
+configure:4280: result: yes
+configure:4280: checking for locale.h
+configure:4280: result: yes
+configure:4291: checking for LC_MESSAGES
+configure:4307: gcc -o conftest -g -O2   conftest.c  >&5
+configure:4307: $? = 0
+configure:4315: result: yes
+configure:4332: checking libintl.h usability
+configure:4332: gcc -c -g -O2  conftest.c >&5
+configure:4332: $? = 0
+configure:4332: result: yes
+configure:4332: checking libintl.h presence
+configure:4332: gcc -E  conftest.c
+configure:4332: $? = 0
+configure:4332: result: yes
+configure:4332: checking for libintl.h
+configure:4332: result: yes
+configure:4340: checking for ngettext in libc
+configure:4358: gcc -o conftest -g -O2   conftest.c  >&5
+configure:4358: $? = 0
+configure:4367: result: yes
+configure:4371: checking for dgettext in libc
+configure:4389: gcc -o conftest -g -O2   conftest.c  >&5
+configure:4389: $? = 0
+configure:4398: result: yes
+configure:4405: checking for bind_textdomain_codeset
+configure:4405: gcc -o conftest -g -O2   conftest.c  >&5
+configure:4405: $? = 0
+configure:4405: result: yes
+configure:4681: checking for msgfmt
+configure:4708: result: /usr/bin/msgfmt
+configure:4719: checking for dcgettext
+configure:4719: gcc -o conftest -g -O2   conftest.c   >&5
+conftest.c:48: warning: conflicting types for built-in function 'dcgettext'
+configure:4719: $? = 0
+configure:4719: result: yes
+configure:4729: checking if msgfmt accepts -c
+configure:4744: $MSGFMT -c -o /dev/null conftest.foo
+configure:4747: $? = 0
+configure:4749: result: yes
+configure:4759: checking for gmsgfmt
+configure:4790: result: /usr/bin/msgfmt
+configure:4800: checking for xgettext
+configure:4827: result: /usr/bin/xgettext
+configure:4846: gcc -o conftest -g -O2   conftest.c   >&5
+configure:4846: $? = 0
+configure:4928: checking for catalogs to be installed
+configure:4953: result:  bn_IN bn ca cs da de el es fi fr gl gu hi hr hu is it ja ko ml ms nb nl or pa pl pt pt_BR ru sk sl sr@latin sr sv ta tr uk zh_CN zh_TW
+configure:4986: checking whether NLS is requested
+configure:4995: result: yes
+configure:5023: checking for intltool-update
+configure:5056: result: no
+configure:5063: checking for intltool-merge
+configure:5096: result: no
+configure:5103: checking for intltool-extract
+configure:5136: result: no
+configure:5142: error: The intltool scripts were not found. Please install intltool.
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_c_compiler_gnu=yes
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_PYTHON_set=
+ac_cv_env_PYTHON_value=
+ac_cv_env_build_alias_set=set
+ac_cv_env_build_alias_value=x86_64-linux-gnu
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_func_bind_textdomain_codeset=yes
+ac_cv_func_dcgettext=yes
+ac_cv_header_inttypes_h=yes
+ac_cv_header_libintl_h=yes
+ac_cv_header_locale_h=yes
+ac_cv_header_memory_h=yes
+ac_cv_header_stdc=yes
+ac_cv_header_stdint_h=yes
+ac_cv_header_stdlib_h=yes
+ac_cv_header_string_h=yes
+ac_cv_header_strings_h=yes
+ac_cv_header_sys_stat_h=yes
+ac_cv_header_sys_types_h=yes
+ac_cv_header_unistd_h=yes
+ac_cv_objext=o
+ac_cv_path_EGREP='/bin/grep -E'
+ac_cv_path_GMSGFMT=/usr/bin/msgfmt
+ac_cv_path_GREP=/bin/grep
+ac_cv_path_MSGFMT=/usr/bin/msgfmt
+ac_cv_path_PYTHON=/usr/bin/python
+ac_cv_path_XGETTEXT=/usr/bin/xgettext
+ac_cv_path_install='/usr/bin/install -c'
+ac_cv_path_mkdir=/bin/mkdir
+ac_cv_prog_AWK=gawk
+ac_cv_prog_CPP='gcc -E'
+ac_cv_prog_ac_ct_CC=gcc
+ac_cv_prog_cc_c89=
+ac_cv_prog_cc_g=yes
+ac_cv_prog_make_make_set=yes
+am_cv_CC_dependencies_compiler_type=none
+am_cv_python_platform=linux2
+am_cv_python_pyexecdir='${exec_prefix}/lib/python2.6/dist-packages'
+am_cv_python_pythondir='${prefix}/lib/python2.6/dist-packages'
+am_cv_python_version=2.6
+am_cv_val_LC_MESSAGES=yes
+gt_cv_func_dgettext_libc=yes
+gt_cv_func_dgettext_libintl=no
+gt_cv_func_ngettext_libc=yes
+gt_cv_have_gettext=yes
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL='${SHELL} /home/vanmeeuwen/pykolab/missing --run aclocal-1.11'
+ALL_LINGUAS='bn_IN bn ca cs da de el es fi fr gl gu hi hr hu is it ja ko ml ms nb nl or pa pl pt pt_BR ru sk sl sr@latin sr sv ta tr uk zh_CN zh_TW'
+AMDEPBACKSLASH=''
+AMDEP_FALSE=''
+AMDEP_TRUE='#'
+AMTAR='$${TAR-tar}'
+AM_DEFAULT_VERBOSITY=''
+AUTOCONF='${SHELL} /home/vanmeeuwen/pykolab/missing --run autoconf'
+AUTOHEADER='${SHELL} /home/vanmeeuwen/pykolab/missing --run autoheader'
+AUTOMAKE='${SHELL} /home/vanmeeuwen/pykolab/missing --run automake-1.11'
+AWK='gawk'
+CATALOGS=' bn_IN.gmo bn.gmo ca.gmo cs.gmo da.gmo de.gmo el.gmo es.gmo fi.gmo fr.gmo gl.gmo gu.gmo hi.gmo hr.gmo hu.gmo is.gmo it.gmo ja.gmo ko.gmo ml.gmo ms.gmo nb.gmo nl.gmo or.gmo pa.gmo pl.gmo pt.gmo pt_BR.gmo ru.gmo sk.gmo sl.gmo sr@latin.gmo sr.gmo sv.gmo ta.gmo tr.gmo uk.gmo zh_CN.gmo zh_TW.gmo'
+CATOBJEXT='.gmo'
+CC='gcc'
+CCDEPMODE='depmode=none'
+CFLAGS='-g -O2'
+CPP='gcc -E'
+CPPFLAGS=''
+CYGPATH_W='echo'
+DATADIRNAME='share'
+DATESTAMP=''
+DEFS=''
+DEPDIR='.deps'
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP='/bin/grep -E'
+ENTERPRISE_FALSE=''
+ENTERPRISE_TRUE=''
+EXEEXT=''
+GETTEXT_PACKAGE='pykolab'
+GMOFILES=' bn_IN.gmo bn.gmo ca.gmo cs.gmo da.gmo de.gmo el.gmo es.gmo fi.gmo fr.gmo gl.gmo gu.gmo hi.gmo hr.gmo hu.gmo is.gmo it.gmo ja.gmo ko.gmo ml.gmo ms.gmo nb.gmo nl.gmo or.gmo pa.gmo pl.gmo pt.gmo pt_BR.gmo ru.gmo sk.gmo sl.gmo sr@latin.gmo sr.gmo sv.gmo ta.gmo tr.gmo uk.gmo zh_CN.gmo zh_TW.gmo'
+GMSGFMT='/usr/bin/msgfmt'
+GREP='/bin/grep'
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
+INSTOBJEXT='.mo'
+INTLLIBS=''
+INTLTOOL_CAVES_RULE=''
+INTLTOOL_DESKTOP_RULE=''
+INTLTOOL_DIRECTORY_RULE=''
+INTLTOOL_EXTRACT=''
+INTLTOOL_KBD_RULE=''
+INTLTOOL_KEYS_RULE=''
+INTLTOOL_MERGE=''
+INTLTOOL_OAF_RULE=''
+INTLTOOL_PERL=''
+INTLTOOL_POLICY_RULE=''
+INTLTOOL_PONG_RULE=''
+INTLTOOL_PROP_RULE=''
+INTLTOOL_SCHEMAS_RULE=''
+INTLTOOL_SERVER_RULE=''
+INTLTOOL_SERVICE_RULE=''
+INTLTOOL_SHEET_RULE=''
+INTLTOOL_SOUNDLIST_RULE=''
+INTLTOOL_THEME_RULE=''
+INTLTOOL_UI_RULE=''
+INTLTOOL_UPDATE=''
+INTLTOOL_V_MERGE=''
+INTLTOOL_V_MERGE_OPTIONS=''
+INTLTOOL_XAM_RULE=''
+INTLTOOL_XML_NOMERGE_RULE=''
+INTLTOOL_XML_RULE=''
+INTLTOOL__v_MERGE_0=''
+INTLTOOL__v_MERGE_=''
+LDFLAGS=''
+LIBOBJS=''
+LIBS=''
+LN_S=''
+LTLIBOBJS=''
+MAINT='#'
+MAINTAINER_MODE_FALSE=''
+MAINTAINER_MODE_TRUE='#'
+MAKEINFO='${SHELL} /home/vanmeeuwen/pykolab/missing --run makeinfo'
+MKDIR_P='/bin/mkdir -p'
+MKINSTALLDIRS='./mkinstalldirs'
+MSGFMT='/usr/bin/msgfmt'
+MSGFMT_OPTS='-c'
+MSGMERGE=''
+OBJEXT='o'
+PACKAGE='pykolab'
+PACKAGE_BUGREPORT=''
+PACKAGE_NAME='pykolab'
+PACKAGE_STRING='pykolab 0.4.3'
+PACKAGE_TARNAME='pykolab'
+PACKAGE_URL=''
+PACKAGE_VERSION='0.4.3'
+PATH_SEPARATOR=':'
+POFILES=' bn_IN.po bn.po ca.po cs.po da.po de.po el.po es.po fi.po fr.po gl.po gu.po hi.po hr.po hu.po is.po it.po ja.po ko.po ml.po ms.po nb.po nl.po or.po pa.po pl.po pt.po pt_BR.po ru.po sk.po sl.po sr@latin.po sr.po sv.po ta.po tr.po uk.po zh_CN.po zh_TW.po'
+POSUB='po'
+PO_IN_DATADIR_FALSE=''
+PO_IN_DATADIR_TRUE=''
+PYTHON='/usr/bin/python'
+PYTHON_EXEC_PREFIX='${exec_prefix}'
+PYTHON_PLATFORM='linux2'
+PYTHON_PREFIX='${prefix}'
+PYTHON_VERSION='2.6'
+RELEASE='1'
+SET_MAKE=''
+SHELL='/bin/bash'
+STRIP=''
+USE_NLS='yes'
+VERSION='0.4.3'
+XGETTEXT='/usr/bin/xgettext'
+ac_ct_CC='gcc'
+am__EXEEXT_FALSE=''
+am__EXEEXT_TRUE=''
+am__fastdepCC_FALSE=''
+am__fastdepCC_TRUE='#'
+am__include='include'
+am__isrc=''
+am__leading_dot='.'
+am__nodep=''
+am__quote=''
+am__tar='$${TAR-tar} chof - "$$tardir"'
+am__untar='$${TAR-tar} xf -'
+bindir='${exec_prefix}/bin'
+build_alias='x86_64-linux-gnu'
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+dvidir='${docdir}'
+exec_prefix='NONE'
+host_alias=''
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+install_sh='${SHELL} /home/vanmeeuwen/pykolab/install-sh'
+intltool__v_merge_options_0=''
+intltool__v_merge_options_=''
+libdir='${exec_prefix}/lib'
+libexecdir='${prefix}/lib/pykolab'
+localedir='${datarootdir}/locale'
+localstatedir='/var'
+mandir='${prefix}/share/man'
+mkdir_p='/bin/mkdir -p'
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+pkgpyexecdir='${pyexecdir}/pykolab'
+pkgpythondir='${pythondir}/pykolab'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+pyexecdir='${exec_prefix}/lib/python2.6/dist-packages'
+pythondir='${prefix}/lib/python2.6/dist-packages'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME "pykolab"
+#define PACKAGE_TARNAME "pykolab"
+#define PACKAGE_VERSION "0.4.3"
+#define PACKAGE_STRING "pykolab 0.4.3"
+#define PACKAGE_BUGREPORT ""
+#define PACKAGE_URL ""
+#define PACKAGE "pykolab"
+#define VERSION "0.4.3"
+#define STDC_HEADERS 1
+#define HAVE_SYS_TYPES_H 1
+#define HAVE_SYS_STAT_H 1
+#define HAVE_STDLIB_H 1
+#define HAVE_STRING_H 1
+#define HAVE_MEMORY_H 1
+#define HAVE_STRINGS_H 1
+#define HAVE_INTTYPES_H 1
+#define HAVE_STDINT_H 1
+#define HAVE_UNISTD_H 1
+#define HAVE_LOCALE_H 1
+#define HAVE_LC_MESSAGES 1
+#define HAVE_BIND_TEXTDOMAIN_CODESET 1
+#define HAVE_GETTEXT 1
+#define HAVE_DCGETTEXT 1
+#define ENABLE_NLS 1
+
+configure: exit 1
--- /dev/null
+++ pykolab-0.4.3/po/POTFILES
@@ -0,0 +1,273 @@
+	../[encoding: UTF-8] \
+	.././kolab/__init__.py \
+	.././test-wallace-send.py \
+	.././wallace/__init__.py \
+	.././wallace/future_module_footer.py \
+	.././wallace/future_module_correctsentdate.py \
+	.././wallace/modules.py \
+	.././wallace/future_module_googletranslate.py \
+	.././wallace/future_module_dlp.py \
+	.././wallace/module_optout.py \
+	.././wallace/future_module_resources.py \
+	.././wallace/future_module_freebusy.py \
+	.././wallace/future_module_statistics.py \
+	.././wallace/future_module_bcc.py \
+	.././wallace/future_module_conversations.py \
+	.././tests/__init__.py \
+	.././tests/test-002-pykolab_conf_split.py \
+	.././tests/test-003-conf_get_list.py \
+	.././tests/test-001-import_pykolab_conf.py \
+	.././tests/test-000-import_pykolab.py \
+	.././setup-kolab.py \
+	.././play/migrate_mail_to_uid_prod.py \
+	.././play/detect-object-type.py \
+	.././play/test_submission.py \
+	.././play/test_undelete.py \
+	.././play/kolab-sap/kolab_smtp_access_policy.py \
+	.././play/confmgmt/__init__.py \
+	.././play/confmgmt/augeas.py \
+	.././play/confmgmt/db.py \
+	.././play/confmgmt/model.py \
+	.././play/test_folders.py \
+	.././play/migrate_uid_to_mail.py \
+	.././play/split_message_file.py \
+	.././play/test-kolab-smtp-access-policy-load.py \
+	.././play/test-login-as.py \
+	.././play/test_imap.py \
+	.././play/test.py \
+	.././play/test_imapd.py \
+	.././play/augeas-insert.py \
+	.././play/imap_annotations_test.py \
+	.././play/libkolabxml/todo.py \
+	.././play/libkolabxml/event.py \
+	.././play/libkolabxml/event_rfc822.py \
+	.././play/libkolabxml/contact.py \
+	.././play/libkolabxml/event_imap.py \
+	.././play/persistent_search.py \
+	.././play/migrate_mail_to_uid.py \
+	.././play/noheaderini.py \
+	.././play/flawed_zpush_testing_create_folders.py \
+	.././play/migrate_uid_to_mail_prod.py \
+	.././play/cleanup_acls.py \
+	.././play/persistent_search_kolab_23.py \
+	.././play/strip_many_headers.py \
+	.././play/parallel_persistent_searches.py \
+	.././play/sync_repl_kolab_23.py \
+	.././play/test_socket.py \
+	.././play/sqlalchemy_schemadisplay.py \
+	.././play/test_filter.py \
+	.././play/test_socket_client.py \
+	.././play/test-parse_ldap_uri.py \
+	.././play/roundcube_database.py \
+	.././play/confmgmt.py \
+	.././play/rolequota/__init__.py \
+	.././play/test-output.py \
+	.././play/test_cal_spread.py \
+	.././play/migrate_lowercase_uid.py \
+	.././play/test_kolabxml.py \
+	.././play/unicode-to-ascii.py \
+	.././play/openssl/license.py \
+	.././play/fork.py \
+	.././play/pygpgme/verify.py \
+	.././play/pygpgme/sign.py \
+	.././play/test-entitlements.py \
+	.././play/unicode_test.py \
+	.././play/load_test.py \
+	.././play/conf.py \
+	.././play/roundcube_database/__init__.py \
+	.././play/roundcube_database/users.py \
+	.././play/roundcube_database/identities.py \
+	.././play/push_contacts.py \
+	.././play/effectiverights.py \
+	.././play/sync_client.py \
+	.././play/anon-imap/anon-imap.py \
+	.././play/test_page_control.py \
+	.././play/wap/group.members_list.py \
+	.././play/wap/user.info.py \
+	.././play/wap/roles.list.py \
+	.././play/wap/role.delete.py \
+	.././play/wap/form_value.select_options-preferredlanguage.py \
+	.././play/wap/form_value.list_options-c.py \
+	.././play/wap/domain.info.py \
+	.././play/wap/system.select_domain.py \
+	.././play/wap/domains.capabilities.py \
+	.././play/wap/role.find_by_attribute.py \
+	.././play/wap/group.add.py \
+	.././play/wap/form_value.generate_cn.py \
+	.././play/wap/user_types.list.py \
+	.././play/wap/form_value.generate_uid.py \
+	.././play/wap/role.add.py \
+	.././play/wap/scu.py \
+	.././play/wap/user.delete.py \
+	.././play/wap/group.info.py \
+	.././play/wap/role.capabilities.py \
+	.././play/wap/form_value.generate_password.py \
+	.././play/wap/domains.list.py \
+	.././play/wap/group_types.list.py \
+	.././play/wap/user.edit.py \
+	.././play/wap/form_value.generate_mail.py \
+	.././play/wap/user.add.py \
+	.././play/wap/form_value.select_options-ou.py \
+	.././play/wap/groups.list.py \
+	.././play/wap/form_value.generate_displayname.py \
+	.././play/wap/users.list.py \
+	.././play/wap/role.info.py \
+	.././play/wap/system.capabilities.py \
+	.././play/parse_policy.py \
+	.././testaci.py \
+	.././kwap/kwap/__init__.py \
+	.././kwap/kwap/tests/__init__.py \
+	.././kwap/kwap/tests/functional/__init__.py \
+	.././kwap/kwap/tests/functional/test_authentication.py \
+	.././kwap/kwap/tests/functional/test_root.py \
+	.././kwap/kwap/tests/models/test_auth.py \
+	.././kwap/kwap/tests/models/__init__.py \
+	.././kwap/kwap/templates/__init__.py \
+	.././kwap/kwap/templates/domain/__init__.py \
+	.././kwap/kwap/controllers/__init__.py \
+	.././kwap/kwap/controllers/template.py \
+	.././kwap/kwap/controllers/secure.py \
+	.././kwap/kwap/controllers/domain.py \
+	.././kwap/kwap/controllers/error.py \
+	.././kwap/kwap/controllers/root.py \
+	.././kwap/kwap/model/__init__.py \
+	.././kwap/kwap/model/auth.py \
+	.././kwap/kwap/config/__init__.py \
+	.././kwap/kwap/config/environment.py \
+	.././kwap/kwap/config/app_cfg.py \
+	.././kwap/kwap/config/middleware.py \
+	.././kwap/kwap/websetup/__init__.py \
+	.././kwap/kwap/websetup/schema.py \
+	.././kwap/kwap/websetup/bootstrap.py \
+	.././kwap/kwap/lib/__init__.py \
+	.././kwap/kwap/lib/base.py \
+	.././kwap/kwap/lib/app_globals.py \
+	.././kwap/kwap/lib/helpers.py \
+	.././kwap/kwap/grids/__init__.py \
+	.././kwap/kwap/grids/domains.py \
+	.././kwap/ez_setup/__init__.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/master.mak.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/error.mak.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/index.mak.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/environ.mak.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/about.mak.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/data.mak.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/login.mak.py \
+	.././kwap/data/templates/home/kanarip/devel/src/kolab/pykolab.git/kwap/kwap/templates/authentication.mak.py \
+	.././kwap/setup.py \
+	.././saslauthd.py \
+	.././kolabtest.py \
+	.././test-urllib.py \
+	.././bonnie/__init__.py \
+	.././kolabd.py \
+	.././kolabd/__init__.py \
+	.././kolabd/process.py \
+	.././ext/python/Tools/freeze/bkfile.py \
+	.././ext/python/Tools/freeze/freeze.py \
+	.././ext/python/Tools/freeze/makefreeze.py \
+	.././ext/python/Tools/freeze/checkextensions.py \
+	.././ext/python/Tools/freeze/winmakemakefile.py \
+	.././ext/python/Tools/freeze/makemakefile.py \
+	.././ext/python/Tools/freeze/makeconfig.py \
+	.././ext/python/Tools/freeze/parsesetup.py \
+	.././ext/python/Tools/freeze/checkextensions_win32.py \
+	.././ext/python/Tools/freeze/hello.py \
+	.././pykolab/__init__.py \
+	.././pykolab/tests/tests.py \
+	.././pykolab/tests/__init__.py \
+	.././pykolab/tests/imap/__init__.py \
+	.././pykolab/tests/imap/test_create_mailbox.py \
+	.././pykolab/tests/imap/test_login_admin.py \
+	.././pykolab/tests/imap/test_login.py \
+	.././pykolab/tests/wap/__init__.py \
+	.././pykolab/tests/wap/test_login.py \
+	.././pykolab/auth/__init__.py \
+	.././pykolab/auth/ldap/__init__.py \
+	.././pykolab/auth/ldap/openldap/__init__.py \
+	.././pykolab/auth/ldap/cache.py \
+	.././pykolab/auth/ldap/rhds/__init__.py \
+	.././pykolab/auth/ldap/syncrepl.py \
+	.././pykolab/auth/ldap/msds/__init__.py \
+	.././pykolab/auth/ldap/sunds/__init__.py \
+	.././pykolab/auth/ldap/fds/__init__.py \
+	.././pykolab/auth/sql/__init__.py \
+	.././pykolab/utils.py \
+	.././pykolab/base.py \
+	.././pykolab/conf/__init__.py \
+	.././pykolab/conf/defaults.py \
+	.././pykolab/conf/parser.py \
+	.././pykolab/conf/entitlement.py \
+	.././pykolab/logger.py \
+	.././pykolab/errors.py \
+	.././pykolab/constants.py \
+	.././pykolab/translate.py \
+	.././pykolab/cli/__init__.py \
+	.././pykolab/cli/telemetry/__init__.py \
+	.././pykolab/cli/telemetry/cmd_list_sessions.py \
+	.././pykolab/cli/telemetry/cmd_examine_session.py \
+	.././pykolab/cli/telemetry/cmd_examine_command_issue.py \
+	.././pykolab/cli/telemetry/cmd_expire_sessions.py \
+	.././pykolab/cli/cmd_set_mail.py \
+	.././pykolab/cli/commands.py \
+	.././pykolab/cli/cmd_delete_mailbox.py \
+	.././pykolab/cli/cmd_list_deleted_mailboxes.py \
+	.././pykolab/cli/cmd_set_language.py \
+	.././pykolab/cli/cmd_list_mailboxes.py \
+	.././pykolab/cli/cmd_export_mailbox.py \
+	.././pykolab/cli/cmd_transfer_mailbox.py \
+	.././pykolab/cli/cmd_undelete_mailbox.py \
+	.././pykolab/cli/cmd_sync.py \
+	.././pykolab/cli/cmd_list_domains.py \
+	.././pykolab/cli/cmd_remove_mailaddress.py \
+	.././pykolab/cli/cmd_create_mailbox.py \
+	.././pykolab/plugins/__init__.py \
+	.././pykolab/plugins/defaultfolders/__init__.py \
+	.././pykolab/plugins/dynamicquota/__init__.py \
+	.././pykolab/plugins/recipientpolicy/__init__.py \
+	.././pykolab/wap_client.old/__init__.py \
+	.././pykolab/xml/__init__.py \
+	.././pykolab/xml/2_0/__init__.py \
+	.././pykolab/xml/event.py \
+	.././pykolab/xml/common.py \
+	.././pykolab/xml/1_0/__init__.py \
+	.././pykolab/wap_client/__init__.py \
+	.././pykolab/wap_client/request.py \
+	.././pykolab/wap_client/connect.py \
+	.././pykolab/setup/__init__.py \
+	.././pykolab/setup/setup_zpush.py \
+	.././pykolab/setup/setup_ldap.py \
+	.././pykolab/setup/setup_roundcube.py \
+	.././pykolab/setup/components.py \
+	.././pykolab/setup/setup_mta.py \
+	.././pykolab/setup/setup_imap.py \
+	.././pykolab/setup/setup_kolabd.py \
+	.././pykolab/setup/setup_mysql.py \
+	.././pykolab/setup/setup_freebusy.py \
+	.././pykolab/imap/__init__.py \
+	.././pykolab/imap/dovecot.py \
+	.././pykolab/imap/cyrus.py \
+	.././pykolab/telemetry.py \
+	.././pykolab/tests.old/mail.py \
+	.././pykolab/tests.old/calendar.py \
+	.././pykolab/tests.old/zpush/__init__.py \
+	.././pykolab/tests.old/zpush/test_000_001.py \
+	.././pykolab/tests.old/zpush/test_000_002.py \
+	.././pykolab/tests.old/zpush/test_000_000.py \
+	.././pykolab/tests.old/constants.py \
+	.././pykolab/tests.old/contacts.py \
+	.././pykolab/tests.old/ksap/__init__.py \
+	.././pykolab/tests.old/ksap/test_sendera.py \
+	.././pykolab/tests.old/create-contacts.py \
+	.././wallace.py \
+	.././test-wallace-loadmsg.py \
+	.././kolab.py \
+	.././cyruslib.py \
+	.././test-wallace.py \
+	.././test-search.py \
+	.././conf.py \
+	.././bin/kolab_smtp_access_policy.py \
+	.././bin/kolab_parse_telemetry.py \
+	.././bin/test-read-input.py \
+	.././bin/test_parse_policy.py \
+	.././bonnie.py \
+	.././saslauthd/__init__.py
